core/src/main/java/com/homeaway/streamplatform/streamregistry/service/impl/StreamServiceImpl.java

```diff
@@ -157,17 +157,10 @@ private void applyDefaultPartition(Stream stream) {
      */
     private void verifyAndUpsertTopics(Stream stream, boolean isNewStream) throws StreamCreationException, ClusterNotFoundException {
         List<String> vpcList = stream.getVpcList();
```

Comment:
Might want to put in a comment here and reference the issue....
For future developers to know what's going on here... and the thinking...

Maybe something like:
```
// Only pay attention to "primary" (not replacted) vpcList that way if replicated targets are different
// we do not get "stuck". Reference the appropriate issue (#114) and (#52) that has the the context.
```